src/gui/usbipd_gui.rs

```diff
@@ -150,22 +137,120 @@ impl UsbipdGui {
         self.window.set_visible(false);
     }
 
-    fn show(&self) {
+    fn show(self: &Rc<UsbipdGui>) {
```

Comment:
Any reason for this change? It doesn't look like an `Rc` is needed in this method.